tools/pmd_fdr/pmd_fdr.xml

```diff
@@ -0,0 +1,76 @@
+<tool id="pmd_fdr" name="PMD FDR" version="0.1.0">
+    <description>recalculate FDR fom precursor mass discrepancy</description>
+    <requirements>
+        <requirement type="package" version="">R</requirement>
```

Comment:
r-base and versions are missing

Reply:
ping

Reply:
Thanks for the review.   I will start addressing your points now.